dotnet/samples/TeamsChefBot/GettingStarted.txt

```diff
@@ -0,0 +1,23 @@
+Welcome to Teams Toolkit!
```

Comment:
Open question - use `.md` or `.txt`?

Reply:
Both looks good to me. I just copy it from TTK sample.

Reply:
TTK uses `txt` since it will be automatically opened in VS after creating, and VS does not support to preview markdown by default...
For these samples directly on GitHub, markdown seems to have better UI.

Anyway, let's keep this as open question and get more feedbacks.

Reply:
As the readme gets longer I think it's better to have it in markdown for the better UI. It would be a good idea to show screenshots of a sample bot's capabilities, and that wouldn't be possible with a .txt file.

Reply:
Agree. Updated to `.md`.